raiden/connection_manager.py

```diff
@@ -126,34 +143,48 @@ def leave(self, wait_for_settle=True, timeout=30):
                 c.partner_address) for c in open_channels]
             for channel in channel_specs:
                 try:
-                    self.api.close(*channel),
-                except RuntimeError:
+                    self.api.close(*channel)
+                except:
```

Comment:
Hmmm now I look at it a bit closer. It's conditionally reraised, so we could indeed have unexpected consequences here. Only times where I can think of catch-all excepts and reraising is some special kind of logging and then reraising.

@konradkonrad Would it be difficult to list out the expected exception/s for when the channel is already closed?